pkgs/top-level/all-packages.nix

```diff
@@ -5176,9 +5176,13 @@ in
     inherit (stdenvAdapters) overrideCC;
   };
 
-  llvmPackages_37 = callPackage ../development/compilers/llvm/3.7 {
+  llvmPackages_37 = callPackage ../development/compilers/llvm/3.7 ({
```

Comment:
I'm pretty sure I should do this somewhere else but I couldn't get it working in the stdenv bootstrap phases.